packages/image/src/index.js

```diff
@@ -36,8 +36,9 @@ export default function image(opts = {}) {
         return null;
       }
 
-      const source = readFileSync(id, 'base64');
-      const data = `'data:${mime};base64,${source}'`;
+      const format = mime.includes('svg') ? 'utf-8' : 'base64';
```

Comment:
I think it'd be better to be precise with the mime type.
```suggestion
      const format = mime === 'image/svg+xml' ? 'utf-8' : 'base64';
```
or
```suggestion
      const format = mime === mimeTypes['.svg'] ? 'utf-8' : 'base64';
```

Reply:
Good call here.